index.md

```diff
@@ -1,3 +1,7 @@
+---
+layout: default
```

Comment:
I have a feeling that this is only required with the vanilla Jekyll engine? I haven’t needed it running with the GitHub pages instructions above. But probably doesn’t hurt..

Reply:
I will test it :)

Reply:
I am a Jekyll n00b

Reply:
I left it there, as it works with Jekyll vanilla and the GitHub setup.